<!-- -->
<launch>
    <node pkg="disparity_expansion" type="disparity_conv" name="disparity_conv"/>
    <node pkg="disparity_expansion" type="disparity_expansion_clean" name="disparity_expansion">
        <param name="lut_max_disparity" type="int" value="180" />
        <param name="robot_radius" type="double" value="1.0" />
        <param name="bg_multiplier" type="double" value="2.0" />
        <param name="sensor_pixel_error" type="double" value="0.5" />
        <param name="padding" type="double" value="-1" />
    </node>
</launch>
